.pipelines/e2e.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       set -o pipefail
 
       . secrets/env
-      export HIVE_KUBE_CONFIG_PATH_1="secrets/e2e-aks-kubeconfig_1"
+      export HIVE_KUBE_CONFIG_PATH_1="secrets/aks.kubeconfig"
```

Comment:
Why `HIVE_KUBE_CONFIG_PATH_1` instead of just `HIVE_KUBE_CONFIG_PATH`?

Reply:
@cadenmarchese This changed was made by @petrkotas, you can find full explanation [here](https://github.com/Azure/ARO-RP/pull/2568#discussion_r1037488551) but in short we need it because of hive shards mechanism